### PR TITLE
Add devfile and Workspace icon

### PR DESCRIPTION
**Pull request by tomgeorge**
Signed-off-by: Tom George <tgeorge@redhat.com>

### CODE REVIEW

File: devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+metadata:
+  name: kubernetes-image-puller
+projects:
+  - name: kubernetes-image-puller
+    source:
+      location: 'https://github.com/che-incubator/kubernetes-image-puller.git'
+      type: git
+      branch: master
+    clonePath: src/github.com/redhat-developer/kubernetes-image-puller
```
Comment (ibuziuk) [inline devfile.yaml]: not sure if this line is expected

File: devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+metadata:
+  name: kubernetes-image-puller
+projects:
```
Comment (ibuziuk) [inline devfile.yaml]: we can remove the definition of the project in general e.g. https://github.com/redhat-developer/rh-che/blob/master/devfile.yaml

File: devfile.yaml (hunk 1 above)
Comment (tomgeorge) [reply devfile.yaml]: this clones it into $GOPATH which I think will be expected by the average go user

File: devfile.yaml (hunk 2 above)
Comment (tomgeorge) [reply devfile.yaml]: great idea, changed

**Pull request merged**